### DISCUSSION TITLE
Can anyone share typescript example?

### DISCUSSION THREAD

**Discussion by monzim** [gladly-team/next-firebase-auth#560] [Q&A]
### I am having a problem converting the existing js examples to ts. Please if anyone already implemented it share your repo.
Thanks in advance.**

**Comment by kmjennison** [comment]
The current example has one TypeScript page:
https://github.com/gladly-team/next-firebase-auth/blob/main/example/pages/ssr-no-token.tsx

Hopefully, someone else has a full example they can share.
